doc/cookbook/source/examples/distance/braycurtis.rst

```diff
@@ -0,0 +1,39 @@
+======================
```

Comment:
minor: The "===" are longer than the text, which will cause warnings in the pipeline.